cartography/intel/aws/lambda_function.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_lambda_data(boto3_session, region):
+    """
+    Create an Lambda boto3 client and grab all the lambda functions.
+    """
+    client = boto3_session.client('lambda', region_name=region)
+    paginator = client.get_paginator('list_functions')
+    lambda_functions = []
+    for page in paginator.paginate():
+        for each_function in page['Functions']:
+            lambda_functions.append(each_function)
+    return ({'AWSLambda': lambda_functions})
+
+
+@timeit
+def load_lambda_functions(neo4j_session, data, region, current_aws_account_id, aws_update_tag):
+    ingest_lambda_functions = """
+    MERGE (lambda:AWSLambda{id: {Arn}})
+    ON CREATE SET lambda.firstseen = timestamp()
+    SET lambda.name = {LambdaName},
+    lambda.modifieddate = {LastModified},
+    lambda.arn = {Arn},
+    lambda.runtime = {Runtime},
+    lambda.description = {Description},
+    lambda.timeout = {Timeout},
+    lambda.memory = {MemorySize},
+    lambda.lastupdated = {aws_update_tag}
+    WITH lambda
+    MATCH (role:AWSPrincipal{arn: {Role}})
+    MERGE (lambda)-[r:STS_ASSUME_ROLE_ALLOW]->(role)
+    ON CREATE SET r.firstseen = timestamp()
+    SET r.lastupdated = {aws_update_tag}
+    """
+
+    for lambda_function in data["AWSLambda"]:
```

Comment:
following the changes in line `20`, this should become:
`    for lambda_function in data:`